src/test/java/com/wikia/webdriver/testcases/mobile/mercury/Login.java

```diff
@@ -0,0 +1,66 @@
+package com.wikia.webdriver.testcases.mobile.mercury;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.LoginPage;
+
+import org.testng.annotations.Test;
+
+/**
+ * Created by Qaga on 2015-06-29.
+ */
+public class Login extends NewTestTemplate {
+
+  private static final String
+      ERROR_MESSAGE =
+      "Hm, we don't recognize these credentials. Please try again or register a new account.";
+
+  @Test()
+  public void validUserCanLogIn() {
+    LoginPage loginPage = new LoginPage(driver).get();
+    loginPage.logUserIn(Configuration.getCredentials().userName10,
+                        Configuration.getCredentials().password10);
+
+    Assertion
+        .assertTrue(loginPage.getNav().isUserLoggedIn(Configuration.getCredentials().userName10));
```

Comment:
Don't use Configuration.getCredentials() in every test
below the ERROR_MESSAGE field add:

Credentials credentials = Configuration.getCredentials();

and refer to credentials variable
See how it's done eg. in:
https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/testcases/forumtests/ForumEditModeTests.java